src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -498,4 +506,36 @@ public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkRest
         return new ListResponseModel<OrganizationUserBulkResponseModel>(result.Select(r =>
             new OrganizationUserBulkResponseModel(r.Item1.Id, r.Item2)));
     }
+
+    private async Task HandleOrganizationUserUpdateAsync(OrganizationUser updatedOrganizationUser, Guid savingUserId,
+        OrganizationUserUpdateRequestModel model, int additionalSmSeatsRequired)
+    {
+        if (additionalSmSeatsRequired > 0)
```

Comment:
Invert conditional to reduce nesting. Maybe even move the simple case back up into the endpoint so you only have the private method for the more complex SM update logic.